packages/neuvector/generated-changes/overlay/crds/crd.yaml

```diff
@@ -0,0 +1,1104 @@
+{{- if .Values.crdwebhook.enabled -}}
```

Comment:
Is this file actually how the crd chart is being generated ? I was thinking it was because of the crd folder in upstream: https://github.com/neuvector/neuvector-helm/tree/master/charts/crd

Reply:
This crd is the same one that is available in the core chart itself under template. We are using crd.yaml from the core chart for building the feature chart.

This separate crd chart is useful for the customer who wants to deploy CRs from the CRD before deploying core components.

Reply:
Ah ok, so it actually comes from the overlay then. Which means every time that file changes you'll have to update the overlay.